src/main/java/org/intellij/sdk/codesync/clients/CodeSyncClient.java

```diff
@@ -271,4 +277,20 @@ public JSONObject getTeamActivity(String accessToken) {
         }
     }
 
+    public  static  Boolean  getUserSubscription() {
+        /*
+        - GETs the user subscription
+        */
+        String accessToken = UserAccount.getAccessTokenByEmail();
+        try {
+            JSONResponse response = ClientUtils.sendGet(USER_SUBSCRIPTION_ENDPOINT, accessToken);
+            JSONObject result = response.getJsonResponse();
+            JSONObject subscriptionObject = (JSONObject) result.get("subscription");
+            return Boolean.valueOf(subscriptionObject.get("can_avail_trial").toString());
+        } catch (RequestError | InvalidJsonError | StatusCodeError error) {
+            CodeSyncLogger.error(String.format("Error while getting user subscription. %s", error.getMessage()));
+            return null;
```

Comment:
Since ouput type is boolean, maybe we should return false here?

Reply:
Good idea